Bottom: 366c7831dd7fb4941504754c2759441128ea1e1e
Top:    89480928e0e1e09fa658558e20b161a10d434049
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-07 23:00:57 +0100

action.c: Remove unused parser token F_Scroll ("Scroll")


---

diff --git a/src/action.c b/src/action.c
index 704b98f..cb2a7bf 100644
--- a/src/action.c
+++ b/src/action.c
@@ -164,7 +164,6 @@ typedef enum
   F_Restore,
   F_Rotate,
   F_Save,
-  F_Scroll,
   F_Selected,
   F_SelectedArcs,
   F_SelectedElements,
@@ -400,7 +399,6 @@ static FunctionType Functions[] = {
   {"Revert", F_Revert},
   {"Rotate", F_Rotate},
   {"Save", F_Save},
-  {"Scroll", F_Scroll},
   {"Selected", F_Selected},
   {"SelectedArcs", F_SelectedArcs},
   {"SelectedElements", F_SelectedElements},
